pages/articles/create/index.tsx

```diff
@@ -0,0 +1,264 @@
+              <Button
+                variant="contained"
+                color="secondary"
+                onClick={handleSaveOpen}
+              >
+                Save
+              </Button>
+            </div>
+          </div>
+          <TextField
+            hiddenLabel
+            id="filled-hidden-label-normal"
+            placeholder="Title of article"
+            variant="filled"
+            onChange={(event) => handleTextChange(event)}
+            fullWidth
+            required
+            InputProps={{ disableUnderline: true }}
+          />
+        </div>
+        <div>
+          <h6 style={styledtitle}>
+            Cover Photo<span style={{ color: 'red' }}>*</span>
+          </h6>
+        </div>
+        {
+          <section className="container">
+            <div {...getRootProps({ className: 'dropzone' })}>
+              <input {...getInputProps()} />
+              <Grid
+                sx={styleduploadbox}
+                container
+                direction="column"
+                justifyContent="center"
+                alignItems="center"
+                spacing={2}
+                columns={12}
+              >
+                {file.length !== 0 ? (
+                  <img
+                    src={URL.createObjectURL(file[0])}
+                    height="250px"
+                    onLoad={() => {
+                      URL.revokeObjectURL(URL.createObjectURL(file[0]))
+                    }}
+                  />
+                ) : (
+                  <div>
+                    <Grid
+                      item
+                      xs={12}
+                      sx={{ display: 'flex', justifyContent: 'center' }}
+                    >
+                      <Avatar sx={{ bgcolor: '#bbdefb' }}>
+                        <UploadFileIcon
+                          sx={{
+                            color: '#1976D2',
+                          }}
+                        />
+                      </Avatar>
+                    </Grid>
+                    <Grid item xs={12}>
+                      <p style={{ justifyContent: 'center' }}>
+                        <span
+                          style={{
+                            textDecoration: 'underline',
+                            color: '#1976D2',
+                          }}
+                        >
+                          Click To Upload
+                        </span>
+                        &nbsp;or Drag and Drop
+                      </p>
+                      <p style={{ color: '#9e9e9e' }}>
+                        SVG, PNG, JPG or GIF (max. 3MB)
+                      </p>
+                    </Grid>
+                  </div>
+                )}
+              </Grid>
+            </div>
+          </section>
+        }
+        <div>
+          <h6 style={styledtitle}>Content</h6>
+          <Editor
+            apiKey="qkp2e0rjd9e286x3xoazfsvtniq91z7qm6j6iu65ad0g2o5s"
+            initialValue="<p></p>"
```

Comment:
we need a way to get the value from editor. think there are 2 ways, one is to use a ref but i kept getting type error. the other way is to use it as a controlled component
```
const [content, setContent] = useState('')

<Editor
            apiKey="qkp2e0rjd9e286x3xoazfsvtniq91z7qm6j6iu65ad0g2o5s"
            value={content}
            onEditorChange={(newValue, editor) => setContent(newValue)}
            init={{
              height: 500,
              menubar: true,
              plugins: [
                'highlight',
                'advlist',
                'autolink',
                'lists',
                'link',
                'image',
                'charmap',
                'preview',
                'anchor',
                'searchreplace',
                'visualblocks',
                'code',
                'insertdatetime',
                'media',
                'table',
                'code',
                'help',
                'wordcount',
              ],
              toolbar:
                'undo redo | blocks | ' +
                'bold italic forecolor | alignleft aligncenter ' +
                'alignright alignjustify | bullist numlist outdent indent | ' +
                'removeformat | help',
              content_style:
                'body { font-family:Helvetica,Arial,sans-serif; font-size:14px }',
            }}
          />
```